---
title: "MOLB 7950: Informatics and Statistics for Molecular Biology"
description: >
    A hands-on tutorial of skills and theory needed to process, analyze, and
    visualize output from large biological data sets. We emphasize command-line
    tools, Python programming, and the R statistical computing environment.
---

## Course Overview 

MOLB 7950 teaches bioinforamtic skills and theory needed to process large
biological data sets.

Classes will be held via zoom and recorded using Panopto. Real-time help will be
provided via zoom breakout rooms.

See the [syllabus]({{< ref "syllabus.md" >}}) for dates and times.

The course is divided into blocks:

### Bootcamp

THe **Bootcamp block** covers the basics of shell, R, and Python programming. We
will meet everyday for 90 minutes to cover fundamental concepts you will need
throughout the course.

We will cover the basics of data types you will encounter during biological
data analysis and approaches for their analysis.

### Main blocks

After Bootcamp, will cover experimental approaches used to analyze DNA, RNA, and
protein. Each block spans ~5 weeks, with each week focused on a particular type 
of experiment (see below). Each block covers statistical concepts needed for rigorous
analysis and analysis approaches to process raw data to results (tables or figures).

In most weeks we will discuss and analyze data from a publication. You are 
responsible for reading the week's material before class begins on Monday.

#### Block experiments

1. The **DNA block** covers genome sequencing for identifying mutations, and two
approaches for analyzing chromatin state (ChIP-seq and MNase-seq).

1. The **RNA block** covers RNA-seq, alternative splicing, differential gene
expression, and RNA:protein interactions.

1. The **Protein block** covers mass spectrometry, densitometry, and image
analysis.


## Assignments and Grading

The course measures learning through daily problem sets (graded by your peers),
block exams, a final project, and your participation.

<div class="zebra-striping sane-table">

| **Type**     | **% of grade** |
| :----         | ----:           |
| Problem Sets | 30             |
| Block Exams  | 30             |
| Final Proejct| 20             |
| Particpation | 20             |

</div>

Grades will be assigned as follows:

<div class="zebra-striping sane-table">

| **Percent total points**     | **Grade** |
| ---- | ---- |
| >= 95 | A |
| >= 90 | A- |
| >= 85 | B+ |
| >= 80 | B |

</div>

### Problem sets 

We reinforce concepts with problem sets assigned at the end of class on Mon and
Wed that should take ~30 minutes to complete. Problems sets assigned on Friday
will be more substantial, requiring ~2 hours to complete. Together the problem
sets constitute 30% of your grade.

<div class="zebra-striping sane-table">

| **Assigned** | **Due** | **Grades By** | **Who grades** | **Time to complete (approx)** |
| ----         | ----    | ----           | ----         | ----              |
| Mon @ 12pm   | Tues @ 5pm | Wed @ 5pm  | Instructors / TAs | 60 min |
| Wed @ 12pm   | Thurs @ 5pm | Fri @ 5pm | Instructors | 60 min |
| Fri @ 12pm   | Mon @ 5pm | Wed @ 5pm  | Instructors / TAs | 1-2 hr |

</div>

### Block exams

Block exams will be assigned at the end of the Bootcamp, DNA, RNA, and protein
blocks and should take 3-4 hours to complete. These will be graded by
Instructors / TAs and together the exams contsitute 30% of your grade.

<div class="zebra-striping sane-table">

| **Block exam** | **Assigned** | **Assignment Due** |
| ---            | ---          | ---                |
| DNA exam       | Fri, Oct 2 @ noon | Tues, Oct 6 @ 5 pm |
| RNA exam       | Wed, Nov 11 @ noon | Mon, Nov 16 @ 9 am |
| Protein exam   | Thurs, Dec 3 @ noon | Mon, Dec 7 @ 9 am |

</div>

### Final projects

Final projects can be completed in groups of 1-3 people. Projects will involve
analysis of existing public data sets and end with a short presentation the 
last week of class. The final project constitutes 20% of your grade.

## Teaching team and office hours 

### Instructors

<div class="zebra-striping sane-table">

| **Instructor**        | **E-mail** | **GitHub** | **Schedule a meeting** |
| ----                  | :------:   | :--------------------------: | :----: |
| Jay Hesselberth       | <a href="mailto:jay.hesselberth@cuanschutz.edu"><i class="fa fa-envelope"></i></a> | <a href="http://github.com/jayhesselberth"><i class="fa fa-github"></i></a> | <a href="http://calendly.com/jay-hesselberth"><i class="fa fa-calendar"></i></a> |
| Neel Mukherjee        | <a href="mailto:neelanjan.mukherjee@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/nmukherjee"><i class="fa fa-github"></i></a> | <a href="http://calendly.com/neelanjan-mukherjee"><i class="fa fa-calendar"></i></a> |
| Maggie Lam | <a href="mailto:maggie.lam@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/Maggie-Lam"><i class="fa fa-github"></i></a> | 
| Suja Jagannathan      | <a href="mailto:sujatha.jagannathan@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/sjaganna"><i class="fa fa-github"></i></a> |  <a href="http://calendly.com/jagannas"><i class="fa fa-calendar"></i></a> |
| Srinivas Ramachandran | <a href="mailto:srinivas.ramachandran@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/srinivasramachandran"><i class="fa fa-github"></i></a> |  <a href="http://calendly.com/srinivas-ramachandran"><i class="fa fa-calendar"></i></a> |
| Matt Taliaferro | <a href="mailto:matthew.taliaferro@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/taliaferrojm"><i class="fa fa-github"></i></a> |  <a href="http://calendly.com/matthew-taliaferro"><i class="fa fa-calendar"></i></a> |

</div>

### Teaching Assistants

<div class="zebra-striping sane-table">

| **Instructor**        | **E-mail** | **GitHub** |
| ----                  | :------:   | :--------------------------: |
| Kent Riemondy | <a href="mailto:kent.riemondy@cuanschutz.edu"><i class="fa fa-envelope"></i></a> | <a href="http://github.com/kriemo"><i class="fa fa-github"></i></a> |
| Rui Fu | <a href="mailto:rui.fu@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/raysinensis"><i class="fa fa-github"></i></a> |
| Ryan Sheridan | <a href="mailto:ryan.sheridan@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/sheridar"><i class="fa fa-github"></i></a> |
| Caitlin Winkler | <a href="mailto:caitlin.winkler@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/oligomyeggo"><i class="fa fa-github"></i></a> |
| Tyler Matheny | <a href="mailto:tyler.matheny@cuanschutz.edu"><i class="fa fa-envelope"></i></a> |  <a href="http://github.com/tylermatheny"><i class="fa fa-github"></i></a> |

</div>

## Recommended reading

### Data processing and visualization

- [Fundamentals of Data Visualization](https://serialmentor.com/dataviz/)

### Statistics

- [Modern Statistics for Modern Biology](http://web.stanford.edu/class/bios221/book/)
- [Statistics for Biologists](https://www.nature.com/collections/qghhqm)
- [Think Stats](https://greenteapress.com/wp/think-stats-2e/)
